theSrc/scripts/utils/Utils.spec.js

```diff
@@ -0,0 +1,96 @@
+const _ = require('lodash');
+const Utils = require('./Utils.es6');
+
+describe('Utils:', function () {
+  describe('isNum():', function () {
+    const tests = [
+      { input: 4, expected: true },
+      { input: '4', expected: false },
+      { input: ['4'], expected: false },
+      { input: undefined, expected: false },
+      { input: null, expected: false },
+    ];
+
+    _(tests).each((t) => {
```

Comment:
_(variable) --> I haven't seen this syntax before, is this for const collections in ES6?